app/controllers/images_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ImagesController < ApplicationController
+  before_action :load_user, only: %i(index create)
+
+  def index
+    @image = @user.images.all
+  end
+
+  def create
+    respond_to do |format|
+      if @user.update user_image_params
+        params[:user][:images]["link"].each do |link|
+          @image = @user.images.create!(:link => link)
```

Comment:
```suggestion
          @image = @user.images.create!(link: link)
```
viết theo cú pháp mới đi em

Reply:
Dùng ! như này nếu lỗi thì nó có chạy k?

Reply:
nó trả về exception ạ.